src/main/java/core/basesyntax/services/FileReadService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.services;
+
+import java.util.List;
+
+public interface FileReadService {

```

Comment:
In my opinion, better have one FileService interface with 2 methods for reading and writing.